test/playback.test.js

```diff
@@ -39,7 +46,7 @@ QUnit.module('Playback', {
 
     // uncomment these lines when deugging
     // videojs.log.level('debug');
-    // this.fixture.style = 'position: inherit;';
+    // this.fixture.style.position = 'inherit';
```

Comment:
ie 11 says that `style` is readonly

Reply:
probably works via `setAttribute` but not directly via the prop. Setting the individual css prop sounds good, though.